### PR TITLE
Adding client cert support to helm chart

### LINKED ISSUE #20: Update helm chart to support mutual TLS
In #19 the application supports mutual TLS via two new env vars.

The helm chart hasn't been updated to add support for them.

### PR DESCRIPTION
**Pull request by dexa187**
Fixes #20

### CODE REVIEW

File: helm-chart/fabric-logger/templates/deployment.yaml [hunk 1]
```diff
@@ -134,6 +150,27 @@ spec:
               path: tlscacert.pem
       {{- end }}
 
+      {{- if .Values.secrets.peer.clientKey }}
+      - name: id-clientKey
```
Comment (marcelloromani) [inline helm-chart/fabric-logger/templates/deployment.yaml]: had an error saying volume name must be lowercase
fixed by changing this to `id-clientkey`

`helm install` output:
```
invalid: [spec.template.spec.volumes[4].name: Invalid value: "id-clientKey": a DNS-1123 label must consist of lower case alphanumeric characters or '-',
```

File: helm-chart/fabric-logger/templates/deployment.yaml [hunk 2]
```diff
@@ -134,6 +150,27 @@ spec:
...
+        secret:
+          secretName: {{ .Values.secrets.peer.clientKey }}
+          {{- if .Values.secrets.peer.clientKeyItem }}
+          items:
+            - key: {{ .Values.secrets.peer.clientKeyItem }}
+              path: clientKey.pem
+          {{- end }}
+      {{- end}}
+      {{- if .Values.secrets.peer.clientCert }}
+      - name: id-clientCert
```
Comment (marcelloromani) [inline helm-chart/fabric-logger/templates/deployment.yaml]: same as above, changed to `id-clientcert`

File: helm-chart/fabric-logger/templates/deployment.yaml (hunk 1 above)
Comment (dexa187) [reply helm-chart/fabric-logger/templates/deployment.yaml]: :+1:

File: helm-chart/fabric-logger/templates/deployment.yaml (hunk 2 above)
Comment (dexa187) [reply helm-chart/fabric-logger/templates/deployment.yaml]: :+1:

**Comment by dexa187** [comment]
This PR is now ready.

**Pull request merged**